### ISSUE TITLE
Handle destructuring of this type

### ISSUE THREAD

**Issue by timocov** [timocov/ts-transformer-minify-privates#3]
```ts
class FooBar {
	private privateField: string = 'sda';

	public publicMethod(): void {
		const { privateField } = this; // should correct handle this
		this.privateMethod(privateField);
	}

	private privateMethod(a: string): void {

	}
}
```

**Comment by timocov** [comment]
Test case - 8e190810b4b7dfdb2dc91e446e65e838ef7da039

**Comment by timocov** [comment]
Fixed in f8a9c74aaa407dc0dd4149a300b9b5e86efd50e1
